### PR TITLE
Fix Analytics and rbac

### PR DESCRIPTION
**Pull request by the-redback**

### CODE REVIEW

File: pkg/cmds/root.go [hunk 1]
```diff
@@ -17,20 +17,22 @@ const (
 	gaTrackingCode = "UA-62096468-20"
 )
 
+var analyticsClientID = analytics.ClientID()
```
Comment (aerokite) [inline pkg/cmds/root.go]: Like this
```
var (
    analyticsClientID = analytics.ClientID()
)
```

File: pkg/controller/controller.go [hunk 2]
```diff
@@ -39,6 +39,8 @@ type Options struct {
 	EnableRbac bool
 	//Max number requests for retries
 	MaxNumRequeues int
+	// Analytics Client ID
+	AnalyticsClientID string
```
Comment (aerokite) [inline pkg/controller/controller.go]: Do we need this?

File: pkg/controller/controller.go [hunk 3]
```diff
@@ -39,6 +39,8 @@ type Options struct {
 	EnableRbac bool
```
Comment (aerokite) [inline pkg/controller/controller.go]: We do not need this variable too. Do we?

File: test/e2e/e2e_suite_test.go [hunk 4]
```diff
@@ -23,9 +23,7 @@ import (
 	"k8s.io/client-go/tools/clientcmd"
 )
 
-var storageClass string
-var exporterTag string
-var dockerRegistry string
+var storageClass, exporterTag, dockerRegistry string
```
Comment (tamalsaha) [inline test/e2e/e2e_suite_test.go]: Declare one variable per line. Do like this:
```
var (
 storageClass string
 exporterTag string
 dockerRegistry string
)
```

**Pull request merged**